src/battle/area_flo2/5CE690.c

```diff
@@ -18,9 +18,19 @@ INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218620_5CECB0);
 
 INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218774_5CEE04);
 
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188A8_5CEF38);
-
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188EC_5CEF7C);
+ApiStatus func_802188A8_5CEF38(ScriptInstance* script, s32 isInitialCall) {
+    func_8011D82C(1);
+    *D_801512F0 = 1;
```

Comment:
I forgot what D_801512F0 is defined as, but if it's an array, could you do D_801512F0[0] = 1 instead?

Reply:
It appears to be an `s8*`, and other functions define it directly

Reply:
Gotcha. Okay, nm